chore: finish removing openedx references for enterprise repos

pwnage101 opened the pull request:
As part of ENT-11240 (epic: ENT-11239) for cutting over enterprise IDAs from the `openedx` github org to the `edx` github org, this change codifies the cutover by updating devstack URLs to clone the correct URLs and stop creating fork remotes.  This keeps things simple because we are not actually making an effort to keep the forks in sync.

jmbowman commented on docker-compose-sync.yml:
```diff
@@ -16,6 +16,9 @@ services:
   studio:
     volumes:
       - edxapp-sync:/edx/app/edxapp/edx-platform:nocopy
+  forums:
```
Putting on my DDD "ubiquitous language" hat: we have "forums" here, "forum" above, and our [glossary](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/glossary.html#f) redirects "forum" to "discussion" (and that documentation seems to have standardized on "discussion" throughout).  We should pick one and stick with it.

macdiesel replied:
Should be forum, forgot to change this one.

SalahAdDin commented:
@macdiesel Don't forget add the properly repository for forums.

macdiesel commented:
@SalahAdDin I'm not sure I follow?

SalahAdDin commented:
It works very well, thank you!

pwnage101 committed:
chore: finish removing openedx references for enterprise repos

As part of ENT-11240 (epic: ENT-11239) for cutting over enterprise IDAs
from the `openedx` github org to the `edx` github org, this change
codifies the cutover by updating devstack URLs to clone the correct URLs
and stop creating fork remotes.  This keeps things simple because we are
not actually making an effort to keep the forks in sync.
